spec/unit/action/deploy/environment_spec.rb

```diff
@@ -432,9 +432,25 @@ def initialize(path, info)
 
     let(:mock_stateful_repo_1) { instance_double("R10K::Git::StatefulRepository", :head => "123456") }
     let(:mock_stateful_repo_2) { instance_double("R10K::Git::StatefulRepository", :head => "654321") }
-    let(:mock_git_module_1) { instance_double("R10K::Module::Git", :name => "my_cool_module", :version => "1.0", :repo => mock_stateful_repo_1) }
-    let(:mock_git_module_2) { instance_double("R10K::Module::Git", :name => "my_lame_module", :version => "0.0.1", :repo => mock_stateful_repo_2) }
-    let(:mock_forge_module_1) { double(:name => "their_shiny_module", :version => "2.0.0") }
+    let(:mock_git_module_1) do
+      instance_double("R10K::Module::Git",
+                      :name       => "my_cool_module",
+                      :properties => {
+                        :type     => :git,
+                        :expected => "1.0",
+                        :actual   => mock_stateful_repo_1.head
+                      })
+    end
+    let(:mock_git_module_2) do
+      instance_double("R10K::Module::Git",
+                      :name       => "my_lame_module",
```

Comment:
can we use a different word than `lame`? it has an ableist history.

Reply:
updated